erpnext/public/js/controllers/transaction.js

```diff
@@ -229,6 +229,92 @@ erpnext.TransactionController = erpnext.taxes_and_totals.extend({
 		this.set_dynamic_labels();
 		this.setup_sms();
 		this.setup_quality_inspection();
+		this.setup_barcode_scan();
+		this.frm.$wrapper.find('div.frappe-control.field_barcode p.help-box').text('');
```

Comment:
where is field_bardcode ?

Also bad idea to pick fields via jquery

Reply:
> where is field_bardcode ?
> 
> Also bad idea to pick fields via jquery

the barcode field is a dynamically created field, not accessible via field_dict.